### PR TITLE
Clarifications in monitoring chapter

### PR DESCRIPTION
**Pull request by themr0c**
Reorganize the chapter.
Rewrite the Prometheus part.

### CODE REVIEW

**Review by benoitf — APPROVED** [review]

File: src/main/pages/che-7/administration-guide/assembly_monitoring-che.adoc [hunk 1]
```diff
@@ -15,41 +15,9 @@ summary:
...
-
-ifeval::["{project-context}" == "che"]
-Grafana offers a front-end "facade" with tools to create a various visual representation in the form of dashboards with various panels and graph types.
-endif::[]
-
-Note that this monitoring stack is not an official production-ready solution, but rather has an introduction purpose.
-
-.The structure of {prod-short} monitoring stack
-image::monitoring/monitoring-che-stack-structure.png[link="{imagesdir}/monitoring/monitoring-che-stack-structure.png"]
+This chapter describes how to use  additional tools to process data exposed as metrics by {prod-short}.
```
Comment (benoitf) [inline src/main/pages/che-7/administration-guide/assembly_monitoring-che.adoc]:
```suggestion
This chapter describes how to use additional tools to process data exposed as metrics by {prod-short}. 
```

File: src/main/pages/che-7/administration-guide/assembly_monitoring-che.adoc [hunk 2]
```diff
@@ -15,41 +15,9 @@ summary:
 
 :context: monitoring-che
 
-{prod-short} can expose certain data as metrics, that can be processed by Prometheus
```
Comment (skabashnyuk) [inline src/main/pages/che-7/administration-guide/assembly_monitoring-che.adoc:18]: Can you explain the motivation of removing this information from assembly_monitoring-che.adoc

Comment (themr0c) [reply src/main/pages/che-7/administration-guide/assembly_monitoring-che.adoc:18]: Good point. This chapter should also be about Enabling the production of metrics in Che, indeed. I will make it more clear then.

Comment (themr0c) [reply src/main/pages/che-7/administration-guide/assembly_monitoring-che.adoc:18]: handled feedback from @skabashnyuk - add visibility to enabling and exposing che metrics in last commit

**Comment by themr0c** [comment]
It is not clear to me where to find the `che-monitoring.yaml` necessary to configure Grafana.

I found this file, but as far as I understand, this file is not referencing Grafana: https://github.com/redhat-developer/che-monitoring/blob/master/openshift/che-monitoring.yaml

**Comment by themr0c** [comment]
I will merge this PR as it is. We need more input to get further, but it is already in a better state now.

**Pull request merged**